src/InputField/index.js

```diff
@@ -201,7 +201,7 @@ Suffix.defaultProps = {
 
 export const Input = styled(
   React.forwardRef<Props, HTMLInputElement>(
-    ({ type, size, theme, error, help, inlineLabel, ...props }, ref) => (
+    ({ type, size, error, help, inlineLabel, ...props }, ref) => (
```

Comment:
It was necessary to blacklist `theme` prop, it will be rendered into the DOM otherwise.